### DISCUSSION TITLE
Cofused and need assistane

### DISCUSSION THREAD

**Discussion by thegloballandlady** [community/community#145638] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

how do i turn all the dates to the accurate date format on google spreadsheet, i already tried these formats =text(a1,"mm/dd/yyyy" and =text(datevalue(a1), "mm/dd/yyyy" 

<html>
<body>
<google-sheets-html-origin><style type="text/css"></style>
Order Date
--
10/3/2018
3/2/2018
24-01-2018
27-12-2018
21-08-2018
21-01-2018
16-01-2018
27-03-2018
1/10/2018
1/4/2018
13-11-2018
21-01-2018
30-03-2018
4/2/2018
25-11-2018
25-08-2018
1/12/2018

</body>
</html>

**Comment by ujamX** [comment]
To standardize date formats in Google Spreadsheet, set the column to Format > Number > Date, use the formula =ARRAYFORMULA(IF(A:A<>"", TEXT(DATEVALUE(A:A), "MM/DD/YYYY"), "")), or check your locale settings.

**Comment by thegloballandlady** [comment]
Thank you so much for reaching out
I am so grateful
I appreciate your guidance

**Reply by ujamX** [reply]
Best of luck 🍀
